src/main/modules/sanitize-request/index.ts

```diff
@@ -0,0 +1,32 @@
+import _, { trim } from 'lodash';
+import { strip } from 'node-emoji';
+import sanitizeHtml from 'sanitize-html';
+import { NextFunction } from 'webpack-dev-middleware';
+
+import { AppRequest } from '../../app/controller/AppRequest';
+
+export class SanitizeRequest {
+  private readonly formInputsToOmit = [
+    '_csrf',
+    'onlyContinue',
+    'saveAndComeLater',
+    'onlycontinue',
+    'accessCodeCheck',
+    'submit',
+    'startNow',
+    'goBack',
+    'link',
+  ];
+
+  public sanitizeRequestBody(req: AppRequest, next: NextFunction): void {
+    const sanitizeText = _.flow([strip, sanitizeHtml, _.unescape, trim]);
```

Comment:
is trim working?

Reply:
It works. The trim function was being imported from lodash already, but I've changed it to _.trim.